quandl/api_config.py

```diff
@@ -1,9 +1,16 @@
 class ApiConfig:
     api_key = None
-    api_base = 'https://www.quandl.com/api/v3'
+    api_protocol = 'https://'
+    api_base = '{}www.quandl.com/api/v3'.format(api_protocol)
```

Comment:
What is the advantage of doing it this way?

Reply:
I needed to reference the protocol portion of the url from within the code so it made sense to have it be a variable and then use it to determine the full api url

Reply:
Nice.. and it expects `https://` and not just `https`  as in `urlparse('https://www.quandl.com/api/v3').scheme`

Reply:
I'm not sure if this is a question or not :P